### ISSUE TITLE
browser back button is useless at best if not dangerous in kiosk

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#135]
needs some research to find out how one can handle the back button at all. I am inclined to more or less disable the back / forward buttons but that is of course a rather amateur solution.

**Comment by urapadmin** [comment]
ja, this is indeed bad and the browser's back button is never really useful. 
But, not in v1.

**Comment by urapadmin** [comment]
it got much better with #3210 

We might even be able to close this ticket for now.

**Comment by luizaogs** [comment]
Should this be closed now?

**Comment by urapadmin** [comment]
ja, let's do it
